### DISCUSSION TITLE
app-resync equivalent in argocd-cmd-params-cm

### DISCUSSION THREAD

**Discussion by reggie-k** [argoproj/argo-cd#7597] [Q&A] [answered]
I am using ArgoCD 2.1.6, and I have moved to managing the env variables in argocd-cmd-params-cm.
What would be the key equivalent for app-resync env variable in the config map?

**Comment by pasha-codefresh** [comment, marked as answer]
hi @reggie-k 

https://github.com/argoproj/argo-cd/blob/master/manifests/base/application-controller/argocd-application-controller-statefulset.yaml#L24

It is timeout.reconciliation inside argocd-cm
